contents/docs/libraries/node/index.mdx

```diff
@@ -130,71 +130,69 @@ Since [experiments](/docs/experiments/manual) use feature flags, the code for ru
-
--   Capture an event and associate it with a group
+To create a group or update its properties, use `groupIdentify`:
 
 ```node
-client.capture({
-    event: 'some_event',
-    distinctId: 'user_distinct_id',
-    groups: { company: 'company_id_in_your_db' },
+client.groupIdentify({
+  groupType: 'company',
+  groupKey: 'company_id_in_your_db',
+  properties: {
+    name: 'Awesome Inc',
+    employees: 11,
+  },
+  // optional distinct ID to associate event with an existing person
+  distinctId: 'xyz'
 })
 ```
 
--   Update properties on a group
+The `name` is a special property which is used in the PostHog UI for the name of the group. If you don't specify a `name` property, the group ID is used instead.
+
+If the optional `distinctId` parameter is not provided in the group identify call, it defaults to `${groupType}_${groupKey}` (e.g., `$company_company_id_in_your_db` in the example above). This default behavior results in each group appearing as a separate person in PostHog. To avoid this, it's often more practical to use a consistent `distinctId`, such as `group_identifier`.
+
+Once a group is created, you can use the `capture` method and pass in the `groups` parameter to capture an event with group analytics.
 
 ```node
-client.groupIdentify({
-    groupType: 'company',
-    groupKey: 'company_id_in_your_db',
-    properties: {
-        name: 'Awesome Inc',
-        employees: 11,
-    },
-    // optional distinct ID to associate event with an existing person
-    distinctId: 'xyz'
+client.capture({
+  event: 'some_event',
+  distinctId: 'user_distinct_id',
+  groups: { company: 'company_id_in_your_db' },
 })
 ```
 
-The `name` is a special property which is used in the PostHog UI for the name of the group. If you don't specify a `name` property, the group ID will be used instead.
-
-If the optional `distinct_id` is not provided in the group identify call, it defaults to `${groupType}_${groupKey}` (e.g., `$company_company_id_in_your_db` in the example above). This default behavior will result in each group appearing as a separate person in PostHog. To avoid this, it's often more practical to use a consistent `distinct_id`, such as `group_identifier`.
-
 ## GeoIP properties
 
-Before posthog-node v3.0, we added GeoIP properties to all incoming events by default. We also used these properties for feature flag evaluation, based on the IP address of the request. This isn't ideal since they are created based on your server IP address, rather than the user's, leading to incorrect location resolution.
+Before `posthog-node` v3.0, we added GeoIP properties to all incoming events by default. We also used these properties for feature flag evaluation, based on the IP address of the request. This isn't ideal since they are created based on your server IP address, rather than the user's, leading to incorrect location resolution.
```

Comment:
Not related to this PR, but do you think we at some point need to explain GeoIP in a page/guide somewhere?